cli/tests/testdata/npm/cjs_yargs/main.out

```diff
@@ -0,0 +1,54 @@
+Download http://localhost:4545/npm/registry/yargs
```

Comment:
Do we need all of these Download lines? Could we assume that everything is working properly if the `start server on` line is matched?